app/internal/actor/type.go

```diff
@@ -0,0 +1,97 @@
+package actor
```

Comment:
There is some stuff that we could move out of `page/data.go` into this package, so will take a look after this PR.